examples/gift_card/plutus.json

```diff
@@ -0,0 +1,124 @@
+      "hash": "9185bf20c9bbdd7062d9b2dc11a03290bfb256938395d56855baee81"
+    },
+    {
+      "title": "main.redeem",
+      "datum": {
+        "title": "Unit",
+        "description": "The nullary constructor.",
+        "schema": {
+          "anyOf": [
+            {
+              "dataType": "constructor",
+              "index": 0,
+              "fields": []
+            }
+          ]
+        }
+      },
+      "redeemer": {
+        "title": "Unit",
+        "description": "The nullary constructor.",
+        "schema": {
+          "anyOf": [
+            {
+              "dataType": "constructor",
+              "index": 0,
+              "fields": []
+            }
+          ]
+        }
+      },
+      "parameters": [
+        {
+          "title": "token_name",
+          "schema": {
+            "dataType": "bytes"
+          }
+        },
+        {
+          "title": "policy_id",
+          "schema": {
+            "dataType": "bytes"
+          }
+        }
+      ],
+      "compiledCode": "5901980100003232323232323232323232232232222533300a323232323232323232323232533301b301e0021323253330183370e00266e052000480084cdc780100a0a50375a60360046eb8c06400454cc0612401364c6973742f5475706c652f436f6e73747220636f6e7461696e73206d6f7265206974656d73207468616e2069742065787065637465640016301c00132320010013232332232332232533301c3370e00290010801099190009bab30240013012003301e375400400297adef6c603322330050020010020010030103001001222533301c00213374a900125eb804c8c8c8c94ccc06ccdc7802800899ba548000cc084dd300125eb804ccc01c01c00c014dd7180e8019bab301d0023020003301e00237566034002603400260320026030002602e00260080026028002600400644646660020029000001911199980819b870040020172333004004337000069001180c800800918079baa001149858dd70009bae0013001001222533300900214984cc014c004c02c008ccc00c00cc0300080055cd2b9b5738aae7555cf2ab9f5740ae855d11",
+      "hash": "e1c247d17b275dc889a0437d23f7791a2ea60a6d1b84612572668ec2"
+    }
+  ]
+}
```

Comment:
You might want to re-generate that with the new blueprint "algorithm" ☝️

Reply:
done